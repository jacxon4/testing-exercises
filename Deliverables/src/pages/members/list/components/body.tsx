import * as React from 'react';
import { Member } from '../viewModel';
import { Row } from './row';

export interface BodyProps {
  members: Member[];
}

export const Body: React.StatelessComponent<BodyProps> = (props) => (
  <tbody>
    {
      props.members.map((member) => (
        <Row
          key={member.id}
          member={member}
        />
      ))
    }
  </tbody>
);
