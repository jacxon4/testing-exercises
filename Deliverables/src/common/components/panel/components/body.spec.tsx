import * as React from 'react';
import {shallow} from 'enzyme';
import {Body} from './body';

describe('common/components/panel/components/body',()=>{
    it('should render when passing required properites',()=>{
        const component = shallow(<Body>
            <h1>Test childred</h1>
        </Body>);

        expect(component).toMatchSnapshot();

    })

})