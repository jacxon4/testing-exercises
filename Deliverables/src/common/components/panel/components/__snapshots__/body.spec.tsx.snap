// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/components/panel/components/body should render when passing required properites 1`] = `
<ul
  className="list-group list-group-flush"
>
  <li
    className="list-group-item"
  >
    <h1>
      Test childred
    </h1>
  </li>
</ul>
`;
